sw/logalizer/sdlogger_download.c

```diff
@@ -323,11 +330,12 @@ void write_command(float value)
 */
 void parse_single_byte(unsigned char byte)
 {
+//  printf("parser.state %d\n", parser.state);
```

Comment:
maybe better to remove all commented print ?

Reply:
I'd rather leave them in place, I actually found it helpful when testing and trying to understand the code. Ok?

Reply:
I guess we can keep that. Maybe a compile flag to enable them all at once ?

Reply:
Will do